src/dftbp/dftbplus/mainapi.F90

```diff
@@ -85,7 +86,7 @@ subroutine setGeometry(env, main, coords, latVecs, coordOrigin)
         main%origin = [0.0_dp,0.0_dp,0.0_dp]
       end if
     else
-      main%tLatticeChanged = .false.
+      !main%tLatticeChanged = .false.
```

Comment:
That's dangerous, as lattice changes won't be considered! (What was the problem here?)